packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/Buttons/index.tsx

```diff
@@ -27,15 +27,28 @@ const Left = styled.div`
     display: flex;
 `;
 
-const Button = styled.div`
+const Button = styled.div<{ isDisabled?: boolean }>`
```

Comment:
I know it was already in place but I think we should use semantic tags `button` in this case. It might be worth to use existing `button` component from `@trezor/components`